curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/manipulating-complex-objects.md

```diff
@@ -59,13 +59,13 @@ Add a new album to the `myMusic` array. Add `artist` and `title` strings, `relea
 assert(Array.isArray(myMusic));
 ```
 
-`myMusic` should have at least two objects
+`myMusic` should have at least two values
```

Comment:
```suggestion
`myMusic` should have at least two elements
```

Reply:
I feel like entries, items and values are all more or less fine.  I had a quick glance through the curriculum and we use all three.

Reply:
Right, just looking at [Mozilla](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array), they do not use `values` at all to refer to the elements in the array.

Reply:
Elements works for me 🙂

Reply:
Yeah, sorry. Wasn't really worth my bringing up.

Elements are fine.